x/participation/keeper/msg_withdraw_allocations.go

```diff
@@ -12,18 +12,21 @@ import (
 
 func (k msgServer) WithdrawAllocations(goCtx context.Context, msg *types.MsgWithdrawAllocations) (*types.MsgWithdrawAllocationsResponse, error) {
```

Comment:
The response is kind of irrelevant for txs since they makes sense only in blocking mode. I think it's quite usual for responses to be empty, rather we should use events. But we haven't fully addressed that part yet I believe.

Reply:
We can see response format later based on the usage by the front-end. Sometime adding values is practical, the response can be used to describe the state transition that occurs

For some values it necessary: the campaign ID of a new campaign for example since it's generated on-chain. Here the value that be determined before sending the message so it's less important I would say